compiler/src/iree/compiler/Codegen/Common/GPU/GPUVectorDistribution.h

```diff
@@ -16,6 +16,12 @@
 
 namespace mlir::iree_compiler {
 
+constexpr StringLiteral kVectorLayoutFetcherStorageAttrName =
+    "__vector_layout_fetcher_storage";
+
+constexpr StringLiteral kVectorLayoutRedistributeAttrName =
+    "__vector_layout_redistribute";
```

Comment:
these shouldn't be exposed like this. You can just move the setSignatureForRedistribution function to DistributionPattern class.

Reply:
nice, done! :)

Reply:
Did you forget to push this maybe? These shouldn't be exposed to the user, these are internal details.

Reply:
Ah OK, that makes sense, I was mostly replying to `setSignatureForRedistribution` haha. 
Actually done now :)